.docksal/README.md

```diff
@@ -3,13 +3,14 @@
 ##Steps to setup
 
 1. clone repo -- will default SOE-develop-mc branch.
-2. Checkout branch, e.g., `git checkout -b SOE-jiraticket-shortdescription`
+2. Checkout new branch, e.g., `git checkout -b SOE-jiraticket-shortdescription`
 3. Edit the following files, which are located ./docksal
```

Comment:
Maybe note that the developer can know the base_url by the name of the root folder. I.e., if the folder name is grunt-drupal-task, then base_url will be https://grunt-drupal-task.dockal.

Reply:
OK, I'll update the README and clean up merge conflicts.